themes/aframe/source/css/secondary.styl

```diff
@@ -172,6 +172,49 @@ a
 .section-subnav
     margin-bottom 10px
 
+.copy .post__heading
+    margin-bottom 0
+
+.post__projects
+    display flex
+    flex-wrap wrap
+    margin-bottom 15px
+    margin-left: -1px
+    a
+        background-position center
+        background-repeat no-repeat
+        background-size cover
+        border 1px solid transparent
+        height 160px
+        width 50%
+        &:hover
+            box-shadow 0 0 0 1px rgba($copy-link-hover, 0.75)
+
+.post-listing .post__projects a
+    display none
+    &:nth-child(1),
+    &:nth-child(2)
+        display block
+
+.post__projects .blog-attribution
+    border none
+    color $medium
+
+p.post__timestamp
```

Comment:
actually, you don't need `.copy`. just remove the `margin-right 3px` bit too. change to this:

```styl
.post__timestamp
    font-weight 500
```